hack/lib/cpu.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+import os
+
+ncpu = len(os.sched_getaffinity(0)) \
```

Comment:
is this actually more correct? I think this only accounts for core pinning, not CFS quotas?

https://github.com/uber-go/automaxprocs is substantially more complex

Reply:
we can put a tools-module-only main.go for this, at the cost of another dep in the tools module.